variables.tf

```diff
@@ -47,7 +47,7 @@ variable "log_retention_in_days" {
 variable "agents_count" {
   description = "The number of Agents that should exist in the Agent Pool"
```

Comment:
We want to add a description like: Please set `agents_count` `null` while `enable_auto_scaling` is `true` to avoid possible `agents_count` changes.

Reply:
Well, we are not able to ignore change to a variable, but we can ignore the change to the resource variable that are receiving that value:
```hcl
  lifecycle {
    ignore_changes = [default_node_pool["node_count"]]
  }```

What do you think?

Reply:
It'll block users when they want to update the "node_count" while auto_scaling disabled.